From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Michal Sojka <michal.sojka@cvut.cz>
Date: Fri, 27 Dec 2024 21:08:45 +0100
Subject: [PATCH] velodyne_monitor: Remove libcpprest-dev from package.xml

This package is not available in nixpkgs. With this change the
compilation of this package fails, but Nix can at least evaluate it.
---
 package.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/package.xml b/package.xml
index 665e6ab87c..3232c72297 100644
--- a/package.xml
+++ b/package.xml
@@ -14,7 +14,6 @@
   <depend>diagnostic_msgs</depend>
   <depend>diagnostic_updater</depend>
   <depend>fmt</depend>
-  <depend>libcpprest-dev</depend>
   <depend>rclcpp</depend>
   <depend>rclcpp_components</depend>
   <depend>std_msgs</depend>
-- 
2.46.0

